.github/workflows/reusable-changesets-prerelease.yml

```
@@ -0,0 +1,75 @@
+name: "Prerelease Packages"
+on:
+  push:
+    branches:
+      - release/*
+  workflow_call:
+    inputs:
+      app-path:
+        description: "Specifies a base directory to run the task from"
+        required: false
+        default: './'
+        type: string
+      nodejs-version:
+        description: "Specifies which Node.js version to use"
+        required: false
+        default: '16'
+        type: string
+      package-manager:
+        description: "Specifies which package manager to use"
+        required: false
+        default: ''
+        type: string
+      build-command:
+        description: "Specifies a build command used on package finalizing before publish"
+        required: false
+        default: 'build'
+        type: string
+      prerelease-tag:
+        description: "Specifies a tag name that is used for prerelease mode"
+        required: false
+        default: 'pre'
+        type: string
+env:
+  NODE_ENV: production
+  CI: true
+
+jobs:
+  publish:
+    name: Run Changesets Pre release
+    runs-on: [self-hosted, Linux, X64, cache, node-16]
+    defaults:
+      run:
+        working-directory: ${{ inputs.app-path }}
+    steps:
+      - name: Use Node.js ${{ inputs.nodejs-version }}
+        run: echo ${{ inputs.nodejs-version }} > .nvmrc && nvm use
+      - name: Determine which package manager to use
+        id: package-manager
+        uses: bucketplace/ci/.github/actions/select-nodejs-pkg-manager@latest
+        with:
+          package-manager: ${{ inputs.package-manager }}
+      - name: Install dependencies
+        run: ${{ steps.package-manager.outputs.install-command }}
+
+      - name: Enter Pre-release Mode
+        run: yarn changeset pre enter ${{inputs.prerelease-tag}}
+
+      - name: Bump version for Pre-release
+        run: yarn changeset version
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Build packages
+        run: yarn ${{ inputs.build-command }}
+
+      - name: Publish Pre-release version
+        run: yarn changeset publish
+
+      - name: Commit
+        uses: EndBug/add-and-commit@v9 # You can change this to use a specific version.
+        with:
+          add: '.'
+          committer_name: bp-deployer
+          committer_email: bp-deployer@users.noreply.github.com
+          message: '[skip actions] Pre release ${{inputs.prerelease-tag}}'
```

p4: curly 사이에 숨통을 틔여주세요 🤣 

```suggestion
          message: '[skip actions] Pre release ${{ inputs.prerelease-tag }}'
```